backstop/tests/breadcrumb.html

```diff
@@ -0,0 +1,163 @@
+<!--
+  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+  See LICENSE.md in the project root for license terms and full copyright notice.
+-->
+<!DOCTYPE html>
+<html id="theme">
+  <head>
+    <title>Breadcrumb | iTwinUI</title>
+    <link
+      rel="icon"
+      href="../assets/favicon.ico"
+    />
+    <link
+      rel="stylesheet"
+      href="../../lib/css/all.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/demo.css"
+    />
+    <style>
+      .iui-input {
+        width: 450px;
+      }
+    </style>
+    <script src="../assets/theme.js"></script>
+  </head>
+  <body class="iui-body">
+    <theme-button></theme-button>
+    <h1>Breadcrumb</h1>
+    <hr />
+
+    <section id="demo-default">
+
+      <nav
+        class="iui-breadcrumb"
+        aria-label="Breadcrumb"
+      >
+        <button class="iui-button iui-borderless iui-dropdown">
+          <!-- Use folder.svg from @itwin/itwinui-icons package. -->
+          <svg
+            class="iui-icon"
+            aria-hidden="true"
+            viewBox="0 0 16 16"
+          >
+            <path d="M1,13a1,1,0,0,0,1,1H14a1,1,0,0,0,1-1V5a1,1,0,0,0-1-1H7V3A1,1,0,0,0,6,2H2A1,1,0,0,0,1,3Z" />
+          </svg>
+          <svg
+            class="iui-icon"
+            aria-hidden="true"
+            viewBox="0 0 16 16"
+          >
+            <path
+              d="m4.8067 6h6.3953a.27961.27961 0 0 1 .24043.44321l-3.1736 3.45707a.33969.33969 0 0 1 -.48085 0l-3.2217-3.45707a.26909.26909 0 0 1 .24042-.44321z"
+            />
+          </svg>
+        </button>
+        <ol>
+          <li
+            itemscope
+            itemtype="http://data-vocabulary.org/Breadcrumb"
+          >
+            <a
+              href="#"
+              itemprop="url"
+            >
+              <span itemprop="title">Root</span>
+            </a>
+          </li>
+          <li>
+            <span>&mldr;</span>
+          </li>
+          <li
+            itemscope
+            itemtype="http://data-vocabulary.org/Breadcrumb"
+          >
+            <a
+              href="#"
+              itemprop="url"
+            >
+              <span itemprop="title">Nested</span>
```

Comment:
Need to handle too long strings with ellipsis.

Reply:
Yeah, good idea.  I've added ellipsis truncation and set a `max-width`.

Reply:
Now button does not handle too long strings 😀

Reply:
lol son of a gun! 😅 Fixed it in the last push.
![Screen Shot 2021-05-24 at 9 48 18 AM](https://user-images.githubusercontent.com/849817/119357099-07deff00-bc5c-11eb-9588-477037d1c238.png)

Reply:
Now `<a>` is not truncating. Can you add longer string to test to avoid this again? 😀

Reply:
Yeah,
![image](https://user-images.githubusercontent.com/81580355/119504129-5c759d80-bd74-11eb-819e-ec0bdcd1da5c.png)

Reply:
Now it is not centered 😂 
![image](https://user-images.githubusercontent.com/36186912/119509850-ab720180-bd79-11eb-8b73-a79a073afe52.png)

Reply:
🤬 it should be centered AND truncating after this latest push.